database/db.go

```diff
@@ -38,6 +38,27 @@ func NewDBGetter(reader, writer *gorm.DB) *DBGetter {
 	}
 }
 
+func (getter *DBGetter) HealthCheck() error {
+	if getter.readOnlyDb != getter.readWriteDb {
```

Comment:
the PR looks good, but do you think it's better to create an abstraction for this pattern `if getter.readOnlyDb != getter.readWriteDb`? We are repeating ourselves here, and in `Close()` function.

How about creating an internal function `allDBs` that returns 2 elements if getter.readOnlyDb != getter.readWriteDb, and only 1 element otherwise?

then in HealthCheck() and Close() we can do this:

```
func (getter *DBGetter) HealthCheck() error {
  for _, db := range getter.allDBs() {
     // do ping, or close in case of close()
  }
}
```

Reply:
Great suggestion